lte/gateway/c/core/oai/tasks/nas5g/include/M5GPDUSessionModificationCommand.hpp

```diff
@@ -0,0 +1,54 @@
+/*
+   Copyright 2020 The Magma Authors.
```

Comment:
For new files the year in the licence header should be the current year "2022".

Reply:
Addressed Review Comment